backend/app/app/db/schema.py

```diff
@@ -1,23 +1,56 @@
-from datetime import datetime
+from datetime import datetime, timezone
 
 import sqlalchemy as sa
 from sqlalchemy.orm import relationship
 
 from app.db.base_class import Base
 
 
+def utcnow():
+    return datetime.now(timezone.utc)
+
+
+class DateTimeUTC(sa.types.TypeDecorator):
+    impl = sa.DateTime
+    cache_ok = True
+
+    def process_bind_param(self, value, dialect):
+        # If the value does not have a timezone, it is assumed to be utc
```

Comment:
Why not migrate the data so that this condition doesn't have to remain in the codebase?

Reply:
I will make a small change so that all datetimes that you want to use in database operations must be converted to utc. It will mean that all datetimes used to query the database will be required to be timezone aware, but that was already a soft requirement. Then the timezone value will be used to convert all datetimes to utc before the datetime is used with the database.